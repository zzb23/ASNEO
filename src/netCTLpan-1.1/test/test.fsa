TMDKSELVQKAKLAEQAERYDDMAAAMKAVTEQGHELSNEERNLLSVAYKNVVGARRSSW
RVISSIEQKTERNEKKQQMGKEYREKIEAELQDICNDVLQLLDKYLIPNATQPESKVFYL
KMKGDYFRYLSEVASGDNKQTTVSNSQQAYQEAFEISKKEMQPTHPIRLGLALNFSVFYY
EILNSPEKACSLAKTAFDEAIAELDTLNEESYKDSTLIMQLLRDNLTLWTSENQGDEGDA
GEGEN
