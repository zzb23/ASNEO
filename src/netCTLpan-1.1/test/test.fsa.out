# /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64/bin/netCTLpan -hlaseq -f test.fsa -a HLA-C15:03
# Thu May 24 21:46:46 2018
# User: zbzhang
# PWD : /home/zbzhang/.local/netCTLpan-1.1/test
# Host: Linux ss1 4.4.0-121-generic x86_64
# -hlaseq  -f                   File with full length HLA sequences
# -f       test.fsa             File name with input
# -a       HLA-C15:03           HLA allele
# Command line parameters set to:
#	[-rdir filename]     /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64 Home directory for NetMHpan
#	[-c filename]        /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64/bin/clepred Cleavage prediction code
#	[-t filename]        /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64/bin/tapmat_pred_fsa Tap prediciton code
#	[-m filename]        /home/zbzhang/.local/netCTLpan-1.1/netMHCpan-2.3//netMHCpan MHC binding prediction code
#	[-v]                 0                    Verbose mode
#	[-dirty]             0                    Dirty mode, leave tmp dir+files
#	[-tdir filename]     /tmp                 Temporary directory (Default $$)
#	[-hlaseq filename]   -f                   File with full length HLA sequences
#	[-a line]            HLA-C15:03           HLA allele
#	[-f filename]        test.fsa             File name with input
#	[-s int]             -1                   Sort output on score: 0 [comb], 1 [MHC], 2 [Cle], 3 [TAP] <0 No sort
#	[-l int]             9                    Peptide length [8-11]
#	[-xls]               0                    Save output to xls file
#	[-xlsfile filename]  NetCTLpan_out.xls    Filename for xls dump
#	[-thr float]         -99.900002           Threshold for output
#	[-listMHC]           0                    Print list of alleles included in netMHCpan
#	[-thrfmt filename]   /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64/data/threshold/%s.thr Format for threshold filenames
#	[-wt float]          0.025000             Weight of tap
#	[-wc float]          0.225000             Weight of Clevage
#	[-ethr float]        1.000000             Threshold for epitopes
#	[-version filename]  /home/zbzhang/.local/netCTLpan-1.1/Linux_x86_64/data/version File with version information

# NetCTLpan version 1.1

Error. No FASTA entries read from test.fsa
